modules/statics/src/base.ts

```diff
@@ -730,7 +730,7 @@ export enum UnderlyingAsset {
   USDX = 'usdx',
   USG = 'usg',
   USPX = 'uspx',
-  UST = 'ust',
```

Comment:
Can you update your commit to reflect this as a breaking change?

Reply:
okay